internal/cmd/gocdk/serve.go

```diff
@@ -16,10 +16,213 @@ package main
 
 import (
 	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
 
 	"golang.org/x/xerrors"
 )
 
 func serve(ctx context.Context, pctx *processContext, args []string) error {
-	return xerrors.New("not implemented")
+	f := newFlagSet(pctx, "serve")
+	if err := f.Parse(args); xerrors.Is(err, flag.ErrHelp) {
+		return nil
+	} else if err != nil {
+		return usagef("gocdk serve: %w", err)
+	}
+	if f.NArg() > 1 {
+		return usagef("gocdk serve [BIOME]")
```

Comment:
Consider a flag instead (that defaults to "dev")?

Reply:
yeah, sounds good

Reply:
sounds fine. would the usage then be `gocdk serve --biome BIOME_NAME]` ?

Reply:
Yes.